classes/module/Module.php

```diff
@@ -866,6 +889,21 @@ public function disableDevice($device)
         return true;
     }
 
+    public function isDeviceEnabled(int $device): bool
```

Comment:
```suggestion
    public function isEnabledForDevice(int $device, int $shopId = null): bool
```

And when `$shopId` is null you should fallback on context's shop

Reply:
But then I guess you should add this shop ID in `enableDevice` and `disableDevice`